spec/swagger_schemas/models/statement.rb

```diff
@@ -0,0 +1,70 @@
+STATEMENT = {
+  description: "A financial statement.",
+  type: :object,
+  required: %i[id type attributes],
+  properties: {
+    id: {
+      "$ref": "#/components/schemas/IDAttribute",
+    },
+    type: {
+      description: "The data type.",
+      type: :string,
+      example: "statement",
+      enum: %w[
+        statement
+      ],
+    },
+    attributes: {
+      properties: {
+        month: {
+          description: "The month which appears on the statement in the DfE portal.",
```

Comment:
This confuses me a bit - what's DFE portal? There will be no portal for lead providers to use for MVP of RECT.

Is this the month the financial statement corresponds to?

Also I'm not sure how to actually view these bits in the actual documentation :D